mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+Individual ID,ID associated with a study participant,,,TRUE,,,,HTAN,
+Biospecimen ID,ID associated with a biosample,,,TRUE,,,,HTAN,
+Sex,Biological sex associated with the specimen donor,,,FALSE,,,,,
+Age at Diagnosis,Age at the time of diagnosis expressed in number of days since birth.,,,FALSE,,,,HTAN,
+Biospecimen Tumor Type,"The type of tumor associated with the specimen. Multiple values permitted, comma separated.",,,FALSE,,,,MC2,
+Tumor Subtype,"The subtype related to the scientific determination and investigation, analysis and recognition of the presence and nature of disease, condition, or injury from expressed signs and symptoms of tissue growth resulting from uncontrolled cell proliferation.",,,FALSE,,,,,
+Tumor Grade,"Numeric value to express the degree of abnormality of cancer cells, a measure of differentiation and aggressiveness.",,,FALSE,,,,HTAN,
+Metastasis at Diagnosis,The text term used to describe the extent of metastatic disease present at diagnosis.,,,FALSE,,,,HTAN,
+Last Known Disease Status,Text term that describes the last known state or condition of an individual's neoplasm.,,,FALSE,,,,HTAN,
+Preservation Method,Text term that represents the method used to preserve the sample.,,,FALSE,,,,HTAN,
+Site of Resection or Biopsy,"The text term used to describe the anatomic site of the resection or biopsy of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).",,,FALSE,,,,HTAN,
+Tissue or Organ of Origin,"The text term used to describe the anatomic site of origin, of the patient's malignant disease, as described by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O).",,,FALSE,,,,HTAN,
+Timepoint Label,"Label to identify the time point at which the clinical data or biospecimen was obtained (e.g. Baseline, End of Treatment, Overall survival, Final). NO PHI/PII INFORMATION IS ALLOWED.",,,FALSE,,,,HTAN,
+Health Care Site,The health care site at which the biospecimen sample was collected.,,,FALSE,,,,,
+NanoString GeoMx DSP Spatial Transcriptomics Level 1,Raw sequencing files from GeoMX experiments,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of GeoMx Lab Worksheet File, Synapse ID of GeoMx DSP PKC File, Synapse ID of GeoMx DSP ROI Segment Annotation File, GeoMx DSP Assay Type, Target Depth, Read Length, ROI Name, AOI Name, Software and Version, GeoMx DSP NGS Sequencing Platform, GeoMx DSP NGS Library Selection Method, GeoMx DSP NGS Library Preparation Kit Name, GeoMx DSP Library Preparation Kit Vendor, GeoMx DSP Library Preparation Kit Version",FALSE,,,,,
```

Comment:
The FASTQ (level 1) attributes are mostly the same as requested for the DCC file (level 2), except "Target Depth" and "Read Length".